demos/https/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 # AFR HTTPS demo
 afr_demo_module(https)
 
-afr_set_demo_metadata(ID "HTTPS_DEMO")
+afr_set_demo_metadata(ID "HTTPS_SYNC_DOWNLOAD_DEMO")
 afr_set_demo_metadata(DESCRIPTION "Examples that demonstrate the HTTPS Client")
-afr_set_demo_metadata(DISPLAY_NAME "HTTPS Client S3 download")
+afr_set_demo_metadata(DISPLAY_NAME "HTTPS Client Demos")
```

Comment:
Per discussion with product management we have concluded on this generic name to encompass all of download, upload, synchronous, and asynchronous.